Makefile

```diff
@@ -62,54 +62,66 @@ TEMP_COMPOSE_YML := $(shell $(if $(IS_WIN), (New-TemporaryFile).FullName, mktemp
 SWARM_HOSTS       = $(shell $(DOCKER) node ls --format="{{.Hostname}}" 2>$(if IS_WIN,null,/dev/null))
 
 
-
-create-stack-file: ## Creates stack file for production as $(output_file) e.g. 'make create-stack-file output_file=stack.yaml'
+.PHONY: config
+create-stack-file: config # TODO: deprecate create-stack-file
+config: ## Creates deploy stack file for production as $(output_file) e.g. 'make config output_file=stack.yaml'
 	$(DOCKER_COMPOSE) -f services/docker-compose.yml -f services/docker-compose.prod.yml config > $(output_file)
 
+
 .PHONY: build
-build: .env .build-webclient ## Builds all core service images (user `make rebuild` to build w/o cache)
-	$(DOCKER_COMPOSE) -f services/docker-compose.yml build --parallel $(SERVICES_LIST)
+build: .env ## Builds all core service images (user 'make build-nc' to build w/o cache)
+	# Compiling front-end
+	$(MAKE) -C services/web/client compile
+	# Building services
+	export BUILD_TARGET=production; \
+	$(DOCKER_COMPOSE) -f services/docker-compose.build.yml build --parallel
+
+
+.PHONY: rebuild build-nc
+rebuild: build-nc #TODO: deprecate rebuild
```

Comment:
nc? well.. not sure that is clearer than rebuild;)

Reply:
You have a good eye :-)
I did it for two reasons:
a) help is now alphabetical and shows "build-nc" as a variation of "build".
b) was following the [Makefile XOps blog](https://itnext.io/docker-makefile-x-ops-sharing-infra-as-code-parts-ea6fa0d22946) and he uses there

I will leave ``rebuild`` as an alias. Would that be fine :-)?